### PR TITLE
Add PySide6 patch for QWheelEvent::delta method (SG-31831)

### PR DESCRIPTION
**Pull request by sansrem**
the QWheelEvent::delta method is currently used in tk_multi_pythonconsole/python/app/input_widget.py and output_widget.py

### CODE REVIEW

**Review by staceyoue — COMMENTED** [review]
@sansrem just curious what the wheel event does in the ShotGrid Python Console, can you provide a quick screen capture of this?

File: python/tank/util/pyside6_patcher.py [hunk 1]
```diff
@@ -253,6 +253,17 @@ def versionFunctions(self, version_profile=None):
...
         QtGui.QOpenGLContext.versionFunctions = versionFunctions
 
+    @classmethod
+    def _patch_QWheelEvent(cls, QtGui):
+        """Patch QWheelEvent."""
+
+        def delta(self):
+            """Patch the delta method."""
+
+            return self.angleDelta().y()
```
Comment (staceyoue) [inline python/tank/util/pyside6_patcher.py:263]: I would just add a comment here to explain why we patch the `delta` with `QWheelEvent::angleDelta().y()`
```suggestion
            # Use the more common mouse vertical scroll as the delta.
            # Horizontal scroll is ignored, use angleDelta().x() if the horizontal scroll is needed.
            return self.angleDelta().y()
```

**Review by staceyoue — CHANGES_REQUESTED** [review]
LGTM! Just one comment for me

**Review by staceyoue — APPROVED** [review]

**Review by NorberMV — APPROVED** [review]
LGTM

**Review by julien-lang — APPROVED** [review]

**Pull request merged**